memory_profiler.py

```diff
@@ -49,6 +49,28 @@ def unicode(x, *args): return str(x)
     has_psutil = False
 
 
+class MemitResult(object):
```

Comment:
This is of course a matter of personal preference but why not use a `namedtuple` here? i.e.

```python
class MemitResult(namedtuple("MemitResult", [
    "mem_usage", "baseline", ...
]):
    def _repr_pretty_(self, p, cycle):
        ...
```